<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DTSPageSynConfig</class>
 <widget class="QWidget" name="DTSPageSynConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>同步设置设置</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>12</y>
     <width>782</width>
     <height>483</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="3">
     <widget class="QTableView" name="m_tvTable">
      <property name="minimumSize">
       <size>
        <width>780</width>
        <height>450</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>780</width>
        <height>450</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="m_pbCancel">
      <property name="minimumSize">
       <size>
        <width>65</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>65</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="m_pbSave">
      <property name="minimumSize">
       <size>
        <width>65</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>65</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
